catroid/src/main/res/values/strings.xml

```diff
@@ -901,6 +901,7 @@
     <string name="show_password">Show Password</string>
     <string name="register_password_mismatch">Passwords do not match</string>
     <string name="sign_in_error">Error during Sign-In</string>
+    <string name="error_google_plus_sign_in">ERROR Google+ Sign in. Status: %s</string>
```

Comment:
After looking through a lot of string resources, I think there is no value in trying to improve these newly added strings :laughing: So I'll keep my suggestions at a minimum. Feel free to add feedback, especially if you disagree.

`Google Sign-In error. Status: %d`

Reply:
why %d at  end?

Reply:
> why %d at end? string not Integer

Ah, yes! Nice catch!